### ISSUE TITLE
[Feature]: Make app selection setting more visible

### ISSUE THREAD

**Issue by MatchaSoba** [YTVanced/VancedManager#758]
### Suggestion

Allow users to select their desired apps (Youtube Vanced/Vanced Music) directly from the main screen, or at least let the user know at a glance that both Vanced and Vanced Music are available to download from Vanced Manager directly.

### Additional Information

Having used Vanced for a long time, I recently decided to give Vanced Music a try. On the main Manager screen there wasn't a single indication of it, turns out to even see the option to download it you had to go through 2 different submenus (Context menu > Settings > Select your Apps). What's wrong with having both apps displayed on the main screen from the get go?

### Additional checks

- [X] I have checked other feature requests and this is not a duplicate.
- [X] This is a suggestion for Vanced Manager and NOT YouTube Vanced/YouTube Vanced Music/Vanced microG.

**Comment by RealKasumi** [comment]
That screen already exists when you setup Vanced Manager for the first time.

**Comment by MatchaSoba** [comment]
Yes, but considering I installed Vanced Manager years ago and had completely forgotten about said screen, would it not be better to make such a setting more visible to long-time users?

**Comment by m3kamil** [comment]
👍
